app.css

```diff
@@ -3,7 +3,12 @@
  * See LICENSE in the project root for license information.
  */
 a {
```

Comment:
No major comments on the CSS functionality

It would be more readable and maintainable if there were comments grouping related styles together, i.e "log-in styles", "modded office styles", etc etc.